src/dmd/mtype.d

```diff
@@ -2704,6 +2704,19 @@ extern (C++) abstract class Type : ASTNode
         return null;
     }
 
+    /****************
+     * Is this type a pointer to a function?
+     * Returns:
+     *  the function type if it is
+     */
+    final pure inout nothrow @nogc
+    TypeFunction isPtrToFunction()
```

Comment:
Perhaps add this to mtype.h as well. There's bound to be some uses of this pattern in the code generator.

Reply:
okay